2023-09-24 18:30:57 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 18:30:57 database: opening C:/Users/schei/Desktop/szakdoga/thesis/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/szakdoga/thesis/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/szakdoga/thesis/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 18:31:02 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@61076312 failed; length 4096 at 8192 [2.2.220/2]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1528)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1433)
	at org.h2.mvstore.FileStore.submitOrRun(FileStore.java:1402)
	at org.h2.mvstore.FileStore.serializeAndStore(FileStore.java:1433)
	at org.h2.mvstore.FileStore.lambda$storeIt$2(FileStore.java:1377)
	at org.h2.mvstore.FileStore.submitOrRun(FileStore.java:1402)
	at org.h2.mvstore.FileStore.storeIt(FileStore.java:1376)
	at org.h2.mvstore.MVStore.storeNow(MVStore.java:844)
	at org.h2.mvstore.MVStore.store(MVStore.java:809)
	at org.h2.mvstore.MVStore.commit(MVStore.java:778)
	at org.h2.mvstore.MVStore.commit(MVStore.java:770)
	at org.h2.mvstore.tx.TransactionStore.endTransaction(TransactionStore.java:633)
	at org.h2.mvstore.tx.Transaction.commit(Transaction.java:511)
	at org.h2.engine.SessionLocal.commit(SessionLocal.java:678)
	at org.h2.engine.Database.<init>(Database.java:376)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@61076312 failed; length 4096 at 8192 [2.2.220/2]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 34 more
Caused by: org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@61076312 failed; length 4096 at 8192 [2.2.220/2]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.DataUtils.writeFully(DataUtils.java:478)
	at org.h2.mvstore.SingleFileStore.writeFully(SingleFileStore.java:67)
	at org.h2.mvstore.SingleFileStore.writeFully(SingleFileStore.java:28)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:366)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1525)
	... 28 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:883)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.h2.mvstore.DataUtils.writeFully(DataUtils.java:474)
	... 33 more
2023-09-24 18:33:41 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 18:33:41 database: opening C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 18:33:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "MIGRATION_MODEL" not found (this database is empty); SQL statement:
SELECT ID, VERSION FROM MIGRATION_MODEL ORDER BY UPDATE_TIME DESC [42104-220]
2023-09-24 18:33:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOG" not found (this database is empty); SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOG [42104-220]
2023-09-24 18:33:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOGLOCK" not found (this database is empty); SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOGLOCK [42104-220]
2023-09-24 18:33:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOG" not found; SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOG [42102-220]
2023-09-24 18:49:12 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 18:49:12 database: opening C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 18:49:18 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 18:49:18 database: opening C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 18:50:36 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 18:50:36 database: opening C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 18:50:41 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Reading from file sun.nio.ch.FileChannelImpl@653853ef failed at 0 (length -1), read 0, remaining 8192 [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Reading from file sun.nio.ch.FileChannelImpl@653853ef failed at 0 (length -1), read 0, remaining 8192 [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Reading from file sun.nio.ch.FileChannelImpl@653853ef failed at 0 (length -1), read 0, remaining 8192 [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.DataUtils.readFully(DataUtils.java:455)
	at org.h2.mvstore.FileStore.readFully(FileStore.java:719)
	at org.h2.mvstore.SingleFileStore.readFully(SingleFileStore.java:60)
	at org.h2.mvstore.SingleFileStore.readFully(SingleFileStore.java:28)
	at org.h2.mvstore.RandomAccessStore.readStoreHeader(RandomAccessStore.java:191)
	at org.h2.mvstore.FileStore.start(FileStore.java:917)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:289)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.readInternal(FileChannelImpl.java:844)
	at java.base/sun.nio.ch.FileChannelImpl.read(FileChannelImpl.java:821)
	at org.h2.mvstore.DataUtils.readFully(DataUtils.java:441)
	... 23 more
2023-09-24 18:50:41 database: opening C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Reading from file sun.nio.ch.FileChannelImpl@653853ef failed at 0 (length -1), read 0, remaining 8192 [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.DataUtils.readFully(DataUtils.java:455)
	at org.h2.mvstore.FileStore.readFully(FileStore.java:719)
	at org.h2.mvstore.SingleFileStore.readFully(SingleFileStore.java:60)
	at org.h2.mvstore.SingleFileStore.readFully(SingleFileStore.java:28)
	at org.h2.mvstore.RandomAccessStore.readStoreHeader(RandomAccessStore.java:191)
	at org.h2.mvstore.FileStore.start(FileStore.java:917)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:289)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.readInternal(FileChannelImpl.java:844)
	at java.base/sun.nio.ch.FileChannelImpl.read(FileChannelImpl.java:821)
	at org.h2.mvstore.DataUtils.readFully(DataUtils.java:441)
	... 23 more
2023-09-24 18:51:31 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 18:51:31 database: opening C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 18:51:36 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 18:51:36 database: opening C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/szakdoga/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:00:47 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:00:47 database: opening C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:00:53 database: opening C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Reading from file sun.nio.ch.FileChannelImpl@265a2ad5 failed at 641892 (length -1), read 0, remaining 768 [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.DataUtils.readFully(DataUtils.java:455)
	at org.h2.mvstore.FileStore.readFully(FileStore.java:719)
	at org.h2.mvstore.SingleFileStore.readFully(SingleFileStore.java:60)
	at org.h2.mvstore.SingleFileStore.readFully(SingleFileStore.java:28)
	at org.h2.mvstore.SFChunk.readFully(SFChunk.java:35)
	at org.h2.mvstore.Chunk.readBufferForPage(Chunk.java:465)
	at org.h2.mvstore.FileStore.readPage(FileStore.java:1961)
	at org.h2.mvstore.MVStore.readPage(MVStore.java:1022)
	at org.h2.mvstore.MVMap.readPage(MVMap.java:632)
	at org.h2.mvstore.Page$NonLeaf.getChildPage(Page.java:1117)
	at org.h2.mvstore.Cursor.hasNext(Cursor.java:64)
	at org.h2.mvstore.MVStore.scrubMetaMap(MVStore.java:325)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:300)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.readInternal(FileChannelImpl.java:844)
	at java.base/sun.nio.ch.FileChannelImpl.read(FileChannelImpl.java:821)
	at org.h2.mvstore.DataUtils.readFully(DataUtils.java:441)
	... 29 more
2023-09-24 19:01:11 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:01:11 database: opening C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:01:16 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:01:16 database: opening C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:01:41 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:01:41 database: opening C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:01:47 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@776b97c6 failed; length 4096 at 8192 [2.2.220/2]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1528)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1433)
	at org.h2.mvstore.FileStore.submitOrRun(FileStore.java:1402)
	at org.h2.mvstore.FileStore.serializeAndStore(FileStore.java:1433)
	at org.h2.mvstore.FileStore.lambda$storeIt$2(FileStore.java:1377)
	at org.h2.mvstore.FileStore.submitOrRun(FileStore.java:1402)
	at org.h2.mvstore.FileStore.storeIt(FileStore.java:1376)
	at org.h2.mvstore.MVStore.storeNow(MVStore.java:844)
	at org.h2.mvstore.MVStore.store(MVStore.java:809)
	at org.h2.mvstore.MVStore.commit(MVStore.java:778)
	at org.h2.mvstore.MVStore.commit(MVStore.java:770)
	at org.h2.mvstore.tx.TransactionStore.endTransaction(TransactionStore.java:633)
	at org.h2.mvstore.tx.Transaction.commit(Transaction.java:511)
	at org.h2.engine.SessionLocal.commit(SessionLocal.java:678)
	at org.h2.engine.Database.<init>(Database.java:376)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@776b97c6 failed; length 4096 at 8192 [2.2.220/2]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 34 more
Caused by: org.h2.mvstore.MVStoreException: Writing to sun.nio.ch.FileChannelImpl@776b97c6 failed; length 4096 at 8192 [2.2.220/2]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.DataUtils.writeFully(DataUtils.java:478)
	at org.h2.mvstore.SingleFileStore.writeFully(SingleFileStore.java:67)
	at org.h2.mvstore.SingleFileStore.writeFully(SingleFileStore.java:28)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:366)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1525)
	... 28 more
Caused by: java.nio.channels.ClosedChannelException
	at java.base/sun.nio.ch.FileChannelImpl.ensureOpen(FileChannelImpl.java:159)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:854)
	at org.h2.mvstore.DataUtils.writeFully(DataUtils.java:474)
	... 33 more
2023-09-24 19:06:31 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:06:31 database: opening C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:06:38 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:06:38 database: opening C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:07:39 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:07:39 database: opening C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:07:44 database: opening C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Reading from file sun.nio.ch.FileChannelImpl@18cdfe68 failed at 636972 (length -1), read 0, remaining 768 [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.DataUtils.readFully(DataUtils.java:455)
	at org.h2.mvstore.FileStore.readFully(FileStore.java:719)
	at org.h2.mvstore.SingleFileStore.readFully(SingleFileStore.java:60)
	at org.h2.mvstore.SingleFileStore.readFully(SingleFileStore.java:28)
	at org.h2.mvstore.SFChunk.readFully(SFChunk.java:35)
	at org.h2.mvstore.Chunk.readBufferForPage(Chunk.java:465)
	at org.h2.mvstore.FileStore.readPage(FileStore.java:1961)
	at org.h2.mvstore.MVStore.readPage(MVStore.java:1022)
	at org.h2.mvstore.MVMap.readPage(MVMap.java:632)
	at org.h2.mvstore.Page$NonLeaf.getChildPage(Page.java:1117)
	at org.h2.mvstore.Cursor.hasNext(Cursor.java:64)
	at org.h2.mvstore.MVStore.scrubMetaMap(MVStore.java:325)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:300)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.readInternal(FileChannelImpl.java:844)
	at java.base/sun.nio.ch.FileChannelImpl.read(FileChannelImpl.java:821)
	at org.h2.mvstore.DataUtils.readFully(DataUtils.java:441)
	... 29 more
2023-09-24 19:08:08 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:08:08 database: opening C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:08:15 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2023-09-24 19:08:15 database: opening C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1134)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db" [90028-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/schei/Desktop/thesis/thesis/KEYCLOAK_SERVER/keycloak-22.0.3/data/h2/keycloakdb.mv.db [2.2.220/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:199)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:171)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:395)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
